selfdrive/loggerd/logger.h

```diff
@@ -15,18 +17,44 @@ const std::string LOG_ROOT = util::getenv_default("HOME", "/.comma/media/0/reald
 
 #define LOGGER_MAX_HANDLES 16
 
+class BZFile {
+ public:
+  BZFile(const char* path) {
+    assert(file = fopen(path, "wb"));
+    int bzerror;
+    bz_file = BZ2_bzWriteOpen(&bzerror, file, 9, 0, 30);
+    assert(bzerror == BZ_OK);
+  }
+  ~BZFile() {
+    int bzerror;
+    BZ2_bzWriteClose(&bzerror, bz_file, 0, nullptr, nullptr);
+    fclose(file);
+  }
+  inline void write(void* data, size_t size) {
```

Comment:
Why is `write` declared as `inline`? This doesn't lead to any performance benefits and should only be done if required for correct linkage.